cypress/e2e/forside.cy.ts

```diff
@@ -19,4 +22,16 @@ describe('Forside', () => {
                 cy.wrap($text).should('not.have.text', '');
             });
     });
+
+    it('Skal vise error dersom man prøver å starte søknad uten å ha avhuket bekreftelsesboks. Når den er avhuket skal man kunne navigere til "Om deg" siden', () => {
+        cy.get('[data-testid="start-søknad-knapp"]').click();
+        cy.get('[data-testid="bekreftelsesboks-container"]');
+        cy.find('.bekreftCheckboksPanel').should('have.class', 'bekreftCheckboksPanel--error');
```

Comment:
Denne testen vil måtte endres dersom man endrer komponenten for panelet, så den er ikke så robust som man gjerne skulle ønske. Men vi har kanskje ikke noen annen måte å se i DOM/Cypress at man får en feilmelding? Eller hva med å se om `bekreftelsesboksFeilmelding`-teksten er tilstede inni `[data-testid="bekreftelsesboks-container"]`